build.sbt

```diff
@@ -151,19 +154,40 @@ lazy val iam = project
                                 scalaTest   % Test)
   )
 
-lazy val shaclValidator = project
-  .in(file("modules/ld/shacl-validator"))
-  .dependsOn(types)
+lazy val indexTypes = project
```

Comment:
could we rename this to `queryTypes`, place it in `modules/query-types` and adjust packages to conform to the new name?

also, please move the elastic search and sparql clients top level.